src/mtx3/commands/downlink/getHalfHourDemandVare.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Downlink command to get reactive energy (`A+R-`) in half hours by date.
+ *
+ * @packageDocumentation
+ *
+ * @example
+ * ```js
+ * import * as getHalfHourDemandVare from 'jooby-codec/mtx3/commands/downlink/getHalfHourDemandVare.js';
```

Comment:
```suggestion
 * import * as getHalfHourDemandVARe from 'jooby-codec/mtx3/commands/downlink/getHalfHourDemandVARe.js';
```

Vare - not a simple word, it's VARe or VArE or VARE (volt-ampere reactive effective)

Reply:
as far as I remember we usually don't maintain upper case in abbreviations

Reply:
applied